test/functional/tests/basic/start_cas.py

```diff
@@ -23,7 +24,8 @@ def test_start_cache_add_core():
           - Cache started successfully.
           - Core added successfully.
     """
-    with TestRun.step("Prepare cache and core devices."):
+
+    with TestRun.step("Prepare cache and core."):
```

Comment:
there are distinctions between cache/cache device and core/core device - in this step the test prepares devices for cache and core, the actual cache and core are set up in the following steps